restructuredtext_lint/test/test.py

```diff
@@ -169,6 +170,12 @@ def test_rst_lint_correct_file(self):
         output = str(raw_output)
         self.assertEqual(output, '')
 
+    def test_rst_lint_folder(self):
+        """The `rst-lint` command should print errors in the files inside the folder."""
+        output = subprocess.check_output((sys.executable, rst_lint_path, f_rst), universal_newlines=True)
+        output = str(output)
+        self.assertEqual(output, '')
```

Comment:
This doesn't seem to be asserting that any files in the folder are failing. Can we verify that it indeed checks the file contents?